@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf

```diff
@@ -112,6 +112,11 @@ if (_velXY > 8.23 && _velXY < 12.35) then {
     setCamShakeParams [0.0, 0.5, 0.0, 0.0, true];
     addCamShake       [2.5, 1, 5];
     enableCamShake false;
+
+    SetCustomSoundController[_heli, "CustomSoundController3", 6.4];
```

Comment:
You set these custom controller to always be 6.4, could this instead be set in config?

Reply:
Will move once tuning is completed